### ISSUE TITLE
Remove *.ts files from src during publishing to npm. 

### ISSUE THREAD

**Issue by majo44** [ocombe/ng2-translate#4]
During compilation I have the errors 

```
error TS6059: File '../node_modules/ng2-translate/src/translate.parser.ts' is not under 'rootDir' '../src'. 'rootDir' is expected to contain all sou
rce files.
error TS6059: File '../node_modules/ng2-translate/src/translate.pipe.ts' is not under 'rootDir' '../src'. 'rootDir' is expected to contain all sourc
e files.
error TS6059: File '../node_modules/ng2-translate/src/translate.service.ts' is not under 'rootDir' '../src'. 'rootDir' is expected to contain all so
urce files.
```

Configuration:

```
// tsconfig.json
{
  "compilerOptions": {
    "target": "es5",
    "module": "system",
    "noImplicitAny": true,
    "declaration": true,
    "experimentalDecorators": true,
    "moduleResolution": "node",
    "noEmitHelpers" : true,
    "rootDir" : "src",
    "outDir" : "tmp/es5"
  },

  "exclude": [
    ".git",
    "node_modules",
    "tmp"
  ]
}
```

This is because ng2-translate.d.ts is importing from folder src where there are .d.ts and .ts files, and ts files are loading by default by tsc.

angular2 have a different approach where ts sources are in ts folder.

After removing .ts files from src everything works fine.

**Comment by ocombe** [comment]
Good point, thank you, I will change that for the next release!

**Comment by ocombe** [comment]
This is fixed in 0.1.5, let me know if that works for you
